### DISCUSSION TITLE
skywalking thread pool plug-in,lambda usage issues

### DISCUSSION THREAD

**Discussion by cuixianyu-github** [apache/skywalking#11108] [Q&A] [answered]
The skywalking thread pool plug-in cannot block the lambda writing link, and the whole link cannot be connected. Is there any other plug-in that can block the lambda writing link?

**Comment by wu-sheng** [comment, marked as answer]
block? We don't block anything. 
Lambda is JVM global function, you would never want to block even intercepting it, because your JVM would be overloaded.

**Reply by cuixianyu-github** [reply]
I sent a multithreaded operation using CompletableFuture, but it wasn't intercepted in the trace link

**Reply by wu-sheng** [reply]
It is common as your use cases are not covered. If you like, contribute or implement plugins.
